view/frontend/web/instantsearch.js

```diff
@@ -1,4 +1,4 @@
-requirejs(['algoliaBundle','Magento_Catalog/js/price-utils'], function(algoliaBundle, priceUtils) {
+requirejs(['algoliaBundle', 'transformHit', 'Magento_Catalog/js/price-utils'], function(algoliaBundle, transformHit, priceUtils) {
```

Comment:
Hi @vmalyk. This will cause one extra file to load. Do you know of any way to prevent it?

Reply:
hi @tkrugg , thnx for review. RequireJS loads once that file and for another requires it`s not loaded anymore. The main prop of this way - It makes possible to replace, extend class by Magento mixins. Extra line-load doesn`t block another scripts loading and makes possible to use Magento minification after remove common.js from head and all global  functions.